runtime-decompiler/src/main/java/org/jrd/frontend/frame/nestedJars/NestedJarsView.java

```diff
@@ -0,0 +1,181 @@
+package org.jrd.frontend.frame.nestedJars;
+
+import org.jrd.backend.data.ArchiveManagerOptions;
+import org.jrd.frontend.frame.main.MainFrameView;
+
+import javax.swing.*;
+import javax.swing.border.EtchedBorder;
+import java.awt.*;
+import java.awt.event.ItemEvent;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class NestedJarsView extends JDialog {
+	private JPanel mainPanel;
+	private NestedPanel nestedPanel;
+	private JPanel okCancelPanel;
+	private JButton okButton;
+	private JButton cancelButton;
+
+	public static class NestedPanel extends JPanel {
+
+		public JCheckBox useDefaults;
+		public JTextField textField;
+		public JButton addButton;
+		public JButton removeButton;
+		public JList list;
+		public JScrollPane scrollPane;
+		DefaultListModel dList;
+
+		NestedPanel() {
+			textField = new JTextField();
+
+			dList = new DefaultListModel();
+			list = new JList(dList);
```

Comment:
I'm in temptation to get rid of jlist and add/remove buttons, in favour of single textarea (multilined, split is done on \s+ with newLines counted anyway).
Nice gui havee jidea - you see it as list, but once you click on edit, it changes to textfield, so yo just write.
Imagine, to write "ext1 ext2 extt3....extN" enter/ok is much better then click "ext1" click, click "ext2" click, .... click extN click...
Up to yoiu

Reply:
I'll add check that will split the textfield text by " " and use them as different extensions